app/src/components/login.rs

```diff
@@ -85,18 +85,16 @@ impl CommonComponent<LoginForm> for LoginForm {
                     server_data: res.server_data,
                     credential_finalization: login_finish.message,
                 };
-                self.common.call_backend(
-                    HostService::login_finish,
-                    req,
-                    Msg::AuthenticationFinishResponse,
-                )?;
+                self.common
+                    .call_backend(HostService::login_finish, req, move |r| {
+                        Msg::AuthenticationFinishResponse(r)
```

Comment:
I think you don't need the lambda here, you can leave it as it was (a reference to the enum constructor)

Reply:
I refactored it to a lambda